plugins/modules/proxmox.py

```diff
@@ -1044,282 +676,1059 @@ def main():
+                    update=self.params.get("update"),
+                    force=self.params.get("force"),
+                )
+        elif state == "absent":
+            self.lxc_absent(
+                vmid,
+                hostname,
+                node=self.params.get("node"),
+                timeout=self.params.get("timeout"),
+                purge=self.params.get("purge"),
+            )
+        elif state == "started":
+            self.lxc_started(
+                vmid,
+                hostname,
+                node=self.params.get("node"),
+                timeout=self.params.get("timeout"),
+            )
+        elif state == "stopped":
+            self.lxc_stopped(
+                vmid,
+                hostname,
+                node=self.params.get("node"),
+                timeout=self.params.get("timeout"),
+                force=self.params.get("force"),
+            )
+        elif state == "restarted":
+            self.lxc_restarted(
+                vmid,
+                hostname,
+                node=self.params.get("node"),
+                timeout=self.params.get("timeout"),
+                force=self.params.get("force"),
+            )
+        elif state == "template":
+            self.lxc_to_template(
+                vmid,
+                hostname,
+                node=self.params.get("node"),
+                timeout=self.params.get("timeout"),
+                force=self.params.get("force"),
+            )
+
+    def lxc_present(self, vmid, hostname, node, update, force):
+        self.check_supported_features()
 
-    elif state == 'started':
         try:
-            vm = proxmox.get_vm(vmid)
-            if getattr(proxmox.proxmox_api.nodes(vm['node']), VZ_TYPE)(vmid).status.current.get()['status'] == 'running':
-                module.exit_json(changed=False, vmid=vmid, msg="VM %s is already running" % vmid)
+            lxc = self.get_lxc_resource(vmid, hostname)
+            vmid = vmid or lxc["id"].split("/")[-1]
+            node = node or lxc["node"]
+        except LookupError:
+            lxc = None
+            vmid = vmid or self.get_nextvmid()
+
+        if node is None:
+            raise ValueError(
+                "Argument 'node' is None, but should be found from VMID/hostname or provided."
+            )
 
-            if proxmox.start_instance(vm, vmid, timeout):
-                module.exit_json(changed=True, vmid=vmid, msg="VM %s started" % vmid)
-        except Exception as e:
-            module.fail_json(vmid=vmid, msg="starting of VM %s failed with exception: %s" % (vmid, e))
+        # check if the container exists already
+        if lxc is not None:
+            if update is None:
+                # TODO: Remove deprecation warning in version 11.0.0
+                self.module.deprecate(
+                    msg="The default value of False for 'update' has been deprecated and will be changed to True in version 11.0.0.",
```

Comment:
YAML booleans are lowercase:
```suggestion
                    msg="The default value of false for 'update' has been deprecated and will be changed to true in version 11.0.0.",
```